flixel/util/FlxPath.hx

```diff
@@ -135,16 +166,59 @@ class FlxPath implements IFlxDestroyable
 		return this;
 	}
 	
-	public function start(Nodes:Array<FlxPoint>, Speed:Float = 100, Mode:Int = FlxPath.FORWARD, AutoRotate:Bool = false):FlxPath
+	/**
+	 * Starts movement along specified path.
+	 * 
+	 * @param	Nodes			An optional array of path waypoints. If null then previously added points will be used.
+	 * @param	Speed			The speed at which the object is moving on the path.
+	 * @param	Mode			Path following behavior (like looping, horizontal only, etc).
+	 * @param	AutoRotate		Whether the object's angle should be adjusted to the path angle during path follow behavior.
+	 * 
+	 * @return	This path object.
+	 */
+	public function start(?Nodes:Array<FlxPoint>, Speed:Float = 100, Mode:Int = FlxPath.FORWARD, AutoRotate:Bool = false):FlxPath
+	{
+		nodes = (Nodes != null) ? Nodes : nodes;
+		setPathProperties(Speed, Mode, AutoRotate);
+		restart();
+		return this;
+	}
+	
+	/**
+	 * Just sets nodes for this path, plus returns this path object. Could be useful for chaining.
+	 * @param	Nodes	An array of path waypoints.
+	 * 
+	 * @return	This path object.
+	 */
+	public function setNodes(Nodes:Array<FlxPoint>):FlxPath
+	{
+		_nodes = Nodes;
+		return this;
+	}
+	
+	/**
+	 * Sets path following properties: speed, mode and auto rotation.
+	 * 
+	 * @param	Speed			The speed at which the object is moving on the path.
+	 * @param	Mode			Path following behavior (like looping, horizontal only, etc).
+	 * @param	AutoRotate		Whether the object's angle should be adjusted to the path angle during path follow behavior.
+	 * 
+	 * @return	This path object.
+	 */
+	public function setPathProperties(Speed:Float = 100, Mode:Int = FlxPath.FORWARD, AutoRotate:Bool = false):FlxPath
```

Comment:
What's the use case for this function? It's really just `start()` with one less argument? Or is it mostly about the `restart()` call?